docs/getting_started_guide/Installation_Options/Repository/Installing_SIMP_From_A_Repository.rst

```diff
@@ -112,6 +112,9 @@ Install the SIMP Server
 
    .. code-block:: bash
 
+      $ yum remove -y puppet
```

Comment:
Why were the steps to remove puppet and pupmod-* RPMs added.  We have never had to do this.